src/test/resources/feedback/workermessage1.txt

```diff
@@ -10,15 +10,12 @@ Rating:
 Feedback: Foobar1
 Quality: 1
 Rating: 1
-
 Rating:
 
 Feedback: Foobar2
 Quality: 2
 Rating: 2
-
 Additional feedback: awes0me Message
-
```

Comment:
the end of the file was here a "\n" which is not generated by the message. Just run the test and see what is happening. If you want to change the layout, change it, this pr is about fixing the bug that the message cannot be loaded since the loader uses relative paths based on the $PWD before, which does not work in any way on a productive system, since $PWD could be anything. While doing this change this test was fixed. (The mocks of the database are illegal the records will be null => NPE for a null record).